client/a8c-for-agencies/components/a4a-confirmation-dialog/style.scss

```
@@ -7,6 +7,15 @@
 		@include heading-large;
 	}
 
+	& .dialog__content {
+		h1.a4a-confirmation-dialog__heading {
+			font-size: rem(24px);
```

Since we don't have an equivalent for 24px, we could use 20px:

```suggestion
			@include heading-large;
```